packages/models-library/src/models_library/projects.py

```diff
@@ -84,6 +84,9 @@ class Project(BaseModel):
     # UI front-end setup (SEE projects_ui.py)
     ui: Optional[StudyUI] = None
 
+    # Quality
+    quality: Dict[str, Any] = {}
```

Comment:
any way we could have here something less open than Any? These are all strings right?

Reply:
Is the content of TSR (i.e. quality entry) well defined?  **Can you please provide a schema of the TSR entries?**  Very soon we need to make sense of it when injecting this metadata in the official sparc dataset.